app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -114,25 +113,14 @@ private void listenToModelChanges() {
         attendeeListener.getNotifier()
             .filter(attendeeModelChange -> attendeeModelChange.getAction().equals(BaseModel.Action.UPDATE))
             .map(DatabaseChangeListener.ModelChange::getModel)
-            .subscribe(this::processUpdatedAttendee, Throwable::printStackTrace);
```

Comment:
I know what you are saying, but if you remove attendeeList from presenter, you'll have to add it in the view to perform these operations:

```
attendeeList.clear();
attendeeList.addAll(attendees);
```

which we don't want